composer.json

```diff
@@ -22,6 +22,8 @@
         "mockery/mockery": "^1.4.4",
         "nunomaduro/collision": "^6.1",
         "nunomaduro/larastan": "^2.0",
+        "pestphp/pest": "^1.21",
+        "pestphp/pest-plugin-laravel": "^1.2",
         "phpunit/phpunit": "^9.5.10",
```

Comment:
Can we uninstall phpunit if we're using pest?

Reply:
Good shout 👍 We can let the pest dependency handle the version